variables.tf

```diff
@@ -92,6 +85,13 @@ variable "database_port" {
 }
 
 
+variable "default_multipart_chunksize_mb" {
+  type        = number
+  description = "The default maximum size of chunks to use when copying. Can be overridden by collection config."
+  default     = "250"
```

Comment:
"" is not needed in default since it's a number

Reply:
Fixed